cli/helper/context.go

```diff
@@ -0,0 +1,53 @@
+package helper
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/context"
+	"github.com/nspcc-dev/neo-go/pkg/wallet"
+)
+
+// validUntilBlockIncrement is the number of extra blocks to add to an exported transaction
+const validUntilBlockIncrement = 50
+
+// InitParameterContext creates incompletely signed transaction which can used
+// as input to `multisig sign`.
+func InitParameterContext(tx *transaction.Transaction, acc *wallet.Account, outFile string) error {
```

Comment:
It's not just init, it's writing to `outFile`.

Reply:
Fixed